### ISSUE TITLE
Monster LED Strip does not work

### ISSUE THREAD

**Issue by scoobaspeaz** [iRayanKhan/homebridge-tuya#156]
**Checklist**
- [ Y] I have read the common issues wiki page
- [ Y ] I have checked to make sure the plugin is up to date

**Describe the bug**
The plugin will not control my LED strip. I have it configured with the proper key and it shows connected. However anytime I try to do anything through homebridge it doesnt actually do anything to the light itself. If I do something through the Tuya Smart app I see in the home bridge logs with a command 8 error.

Here is what I see:

[11/23/2020, 6:59:35 AM] [TuyaLan] Discovered Kenzies LED Strip (eb03d0c3f622c37495by78) identified as RGBTWLight (3.2)
[11/23/2020, 6:59:35 AM] [TuyaLan] Connected to Kenzies LED Strip
[11/23/2020, 6:59:35 AM] [TuyaLan] Ready to handle Kenzies LED Strip (RGBTWLight:3.2) with signature {}
[TuyaAccessory] Color format for Kenzies LED Strip (3.2) is undetectable.
[7:03 AM]

Plus a ton of these:[TuyaDiscovery] UDP from 192.168.1.233:6666 0x000055aa...0x0000aa55

Command 8 error if I control the device through the Tuya Smart app:
[TuyaAccessory] Heard back from Kenzies LED Strip with command 8

Schema for it:

Code | Type | Values
-- | -- | --
switch_led | Boolean | {true,false}
work_mode | Enum | {"range":["white","colour","scene","music"]}
bright_value_v2 | Integer | {"min":10,"scale":0,"unit":"","max":1000,"step":1}
colour_data_v2 | Json | {}
scene_data_v2 | Json | {}
countdown_1 | Integer | {"unit":"","min":0,"max":86400,"scale":0,"step":1}
control_data | Json | {}

**Comment by iRayanKhan** [comment]
Thank you for providing an organized schma. Could you possibly try to Tuya-Web plugin? It should speed up the process for lights. Thanks.

**Comment by scoobaspeaz** [comment]
> Thank you for providing an organized schma. Could you possibly try to Tuya-Web plugin? It should speed up the process for lights. Thanks.

Yes it works great with the tuya-web app. It's what I'm using right now to control it.
